cypress/integration/userCanSaveRecipeToCookbook.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Save recipe to Cookbook', () => {
+  beforeEach(() => {
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/v1/recipes/1',
+      response: 'fixture:single_recipe.json',
+      status: 200
+    })
+
```

Comment:
Can you remove the space here and just use a comma to separate the two routes?

Reply:
done